src/materials/ComputeISoilStress.C

```diff
@@ -23,6 +23,7 @@
 #include "ISoilUtils.h"
 #include "Conversion.h"
 #include "FEProblem.h"
+#include <iostream>
```

Comment:
@somu15 why do you have iostream here again?

Reply:
Apologies, thought I got rid of all iostreams.